modules/ROOT/pages/release-notes.adoc

```diff
@@ -10,6 +10,20 @@ The {bonitaVersion} version is in development.
 
 We are excited to announce the release of xref:applications:bonita-ui-builder.adoc[Bonita UI Builder], our new and enhanced development environment for creating all kind of interfaces, pages and forms for your applications! It brings along a host of advanced features and improved usability, designed to help you create compelling, responsive, and user-friendly interfaces.
 
+Key benefits:
+
+* Rapidly create and deploy UIs to reduce time to market
+* Connect UIs with Bonitasoft for end-to-end alignment
+* Tailor UIs to specific needs with advanced options
+* A drag-and-drop interface empowers all users, regardless of technical skill
+* Built with React for dynamic, responsive design
```

Comment:
Not sure we need to mention React, which is fully hidden to the users. One can have responsive design without React